### PR TITLE
docs: add new ADR reference to OEP52

### PR DESCRIPTION
**Pull request by rgraber**
Adds a reference to a new ADR in openedx-events about allowing multiple event types per topic

### CODE REVIEW

File: oeps/architectural-decisions/oep-0052-arch-event-bus-architecture.rst
Comment (robrap) [inline oeps/architectural-decisions/oep-0052-arch-event-bus-architecture.rst:74]:
```suggestion
  * See `ADR on Multiple Event Types Per Topic`_ to learn more about the requirement of Open edX event bus implementations to support multiple event types per topic.
```

**Review by robrap — COMMENTED** [review]
This would need a modified date and changelog update, and is going to cause a conflict with https://github.com/openedx/open-edx-proposals/pull/477/files. I wonder if we can just add this as a commit to that branch with @navinkarkera's permission?

**Comment by rgraber** [comment]
> This would need a modified date and changelog update, and is going to cause a conflict with https://github.com/openedx/open-edx-proposals/pull/477/files. I wonder if we can just add this as a commit to that branch with @navinkarkera's permission?

Ugh, forgot about changelog. There's no real urgency to this so I can wait until their branch has merged and then rebase.

**Comment by rgraber** [comment]
> > This would need a modified date and changelog update, and is going to cause a conflict with https://github.com/openedx/open-edx-proposals/pull/477/files. I wonder if we can just add this as a commit to that branch with @navinkarkera's permission?
> 
> Ugh, forgot about changelog. There's no real urgency to this so I can wait until their branch has merged and then rebase.

Or better yet I can merge this one and rebase @navinkavera 's

File: oeps/architectural-decisions/oep-0052-arch-event-bus-architecture.rst
Comment (rgraber) [reply oeps/architectural-decisions/oep-0052-arch-event-bus-architecture.rst]: [fixed]

**Review by robrap — APPROVED** [review]

**Pull request merged**
